Rakefile

```diff
@@ -46,9 +46,11 @@ end
 desc 'Grab libs'
 task 'setup_libs' do
   in_venv = system "python -c \"import sys ; exit(not hasattr(sys, 'real_prefix'))\""
-  raise 'Not in dev venv/CI environment - bailing out.' if !in_venv && !ENV['CI']
+  raise 'Not in dev venv/CI environment/Integrations - bailing out.' if !in_venv && !ENV['CI'] && !ENV['SDK_HOME']
```

Comment:
I don't think we set `SDK_HOME` in `dd-agent`, do we? We still need to be able to setup the libs here, so I'm not sure if we can rely on `SDK_HOME` like this. Can you double-check?